draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
-            of A followed by B, C, and D in any order.
-          </t>
-          <figure anchor="ExampleofDefaultDependencyCreation">
-            <name>Example of Default Dependency Creation</name>
-            <artwork type="inline"><![CDATA[
-    A                 A
-   / \      ==>      /|\
-  B   C             B D C
-]]></artwork>
-          </figure>
+        <section anchor="PriorityHistory">
+          <name>Background of Priority in HTTP/2</name>
           <t>
-            An exclusive flag allows for the insertion of a new level of dependencies.  The
-            exclusive flag causes the stream to become the sole dependency of its parent stream,
-            causing other dependencies to become dependent on the exclusive stream.  In the
-            previous example, if stream D is created with an exclusive dependency on stream A, this
-            results in D becoming the dependency parent of B and C.
-          </t>
-          <figure anchor="ExampleofExclusiveDependencyCreation">
-            <name>Example of Exclusive Dependency Creation</name>
-            <artwork type="inline"><![CDATA[
-                      A
-    A                 |
-   / \      ==>       D
-  B   C              / \
-                    B   C
-]]></artwork>
-          </figure>
-          <t>
-            Inside the dependency tree, a dependent stream SHOULD only be allocated resources if
-            either all of the streams that it depends on (the chain of parent streams up to 0x0)
-            are closed or it is not possible to make progress on them.
-          </t>
-          <t>
-            A stream cannot depend on itself.  An endpoint MUST treat this as a <xref target="StreamErrorHandler">stream error</xref> of type <xref target="PROTOCOL_ERROR" format="none">PROTOCOL_ERROR</xref>.
-          </t>
-        </section>
-        <section>
-          <name>Dependency Weighting</name>
-          <t>
-            All dependent streams are allocated an integer weight between 1 and 256 (inclusive).
-          </t>
-          <t>
-            Streams with the same parent SHOULD be allocated resources proportionally based on their
-            weight.  Thus, if stream B depends on stream A with weight 4, stream C depends on stream A
-            with weight 12, and no progress can be made on stream A, stream B ideally receives one-third
-            of the resources allocated to stream C.
-          </t>
-        </section>
-        <section anchor="reprioritize">
-          <name>Reprioritization</name>
-          <t>
-            Stream priorities are changed using the <xref target="PRIORITY" format="none">PRIORITY</xref> frame.  Setting a
-            dependency causes a stream to become dependent on the identified parent stream.
+            HTTP/2 included a rich system for signaling priority of requests.  However, this system
+            proved to be complex and it was not widely implemented.
```

Comment:
Not sure I agree with "not widely implemented"... most H2 implementations have a (basic) version of this. 

"and it's full flexibility was rarely employed" or "and it was rarely correctly implemented" or similar would fit the bill better. (or it can be left out, as this is also said in other terms below)

Reply:
How about s/widely/uniformly ?